tests/bluetooth/host/buf/bt_buf_get_cmd_complete/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+if (NOT BOARD STREQUAL unit_testing)
+  message(FATAL_ERROR "This project can only be used with '-DBOARD=unit_testing'.")
+endif()
+
+FILE(GLOB SOURCES src/*.c)
+
+project(bluetooth_bt_buf_get_cmd_complete)
+
+find_package(Zephyr COMPONENTS unittest HINTS $ENV{ZEPHYR_BASE})
+
+include(${ZEPHYR_BASE}/tests/bluetooth/host/CMakeLists.txt)
```

Comment:
It's my understanding that if we use `include`, we should have a `.cmake` file.
Did you try using libraries instead?

Reply:
Where can I find such guidelines about file names and things to do and things not to do?

I just used one of the suggested ways to name the files by **Torsten**
`CMake files should be named either:
CMakeLists.txt or <propername>.cmake.`

Although I wasn't in favor of using the same file name `CMakeLists.txt ` for all files, but I tried to avoid the dilemma of selecting the proper name.

On the other hand, I tried to build libraries, but It didn't work because it depends on auto generated files while building.
I will try it again when the change made by **Torsten** to use `Kconfig `and `prj.conf `files is merged.

So, for now, do you recommend change the file names again?
If yes, do you think a proper file name can be the parent folder name? (e.g. `host_mocks`, `buf_mocks`, `keys_mocks` ...etc)

Reply:
I'm just parroting what I see on the cmake docs :p 
[Here](https://cmake.org/cmake/help/book/mastering-cmake/chapter/Getting%20Started.html#basic-cmake-usage) it describes that the projects (libraries and executables) will be defined in `CMakeLists.txt` files.
[And here (last snippet)](https://cmake.org/cmake/help/book/mastering-cmake/chapter/Writing%20CMakeLists%20Files.html#variable-scope) we see that they use `lowercasename.cmake` for files that are supposed to be included.